### PR TITLE
Splashscreen configured

### LINKED ISSUE #25: Splashscreen
I would like to work on the Configuration of splash screen according to the design guidelines present in PR #15

### PR DESCRIPTION
**Pull request by abhinavsri360**
### Description
Configured the Splash screen as per Design guidelines. The screenshot is provided below.

Fixes # [ISSUE]
Closes #25 
### How Has This Been Tested?
via running on my android device

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged
- [x] My changes generate no new warnings

Screenshot_20200701-173819

### CODE REVIEW

**Comment by abhinavsri360** [comment]
@himanshusharma89 @CheshtaK Animations added and this page is up and running.

File: lib/constants/strings.dart [hunk 1]
```diff
@@ -2,11 +2,11 @@ class Strings {
   Strings._();
 
   //general
-  static const String app_name = 'bookface';
+  static const String app_name = 'Bookface';
```
Comment (CheshtaK) [inline lib/constants/strings.dart:5]: @abhinavsri360 Let's keep this as before. It's a style we're using for the app name.

File: lib/constants/strings.dart [hunk 2]
```diff
@@ -2,11 +2,11 @@ class Strings {
...
   static const String tagline = 'Do You, uh, Book?';
 
   //login
-  static const String welcome = 'Welcome to bookface';
+  static const String welcome = 'Welcome to Bookface';
```
Comment (CheshtaK) [inline lib/constants/strings.dart:9]: Same goes for this. We could discuss more about it if you feel differently. Feel free to ping me on Slack!

**Review by CheshtaK — CHANGES_REQUESTED** [review]
Could you also provide a gif? The code looks good but I can't test it right now. A gif would be helpful.

**Comment by abhinavsri360** [comment]
@CheshtaK Here's the gif to the animation.
ezgif com-video-to-gif

**Comment by himanshusharma89** [comment]
Looks good to me, @CheshtaK

**Review by CheshtaK — APPROVED** [review]
LGTM

**Pull request merged**